litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx

```diff
@@ -67,8 +67,8 @@ const WorkflowTable: React.FC<WorkflowTableProps> = ({ isCustom }) => {
             expData.push({
               StepIndex: index,
               Name: chaosEngine.metadata.name,
-              Namespace: chaosEngine.spec.appinfo.appns,
-              Application: chaosEngine.spec.appinfo.applabel,
+              Namespace: chaosEngine.spec.appinfo?.appns || '',
```

Comment:
if you are checking for only undefined or null and not 0 or ' ' values `??` might be better here